lldb/test/Shell/SymbolFile/Breakpad/stack-cfi-parsing.test

```diff
@@ -11,7 +11,7 @@ image show-unwind -n func0
 # CHECK-NEXT: This UnwindPlan is sourced from the compiler: yes.
 # CHECK-NEXT: This UnwindPlan is valid at all instruction locations: no.
 # CHECK-NEXT: This UnwindPlan is for a trap handler function: no.
-# CHECK-NEXT: Address range of this UnwindPlan: [stack-cfi-parsing.out..module_image + 0-0x0000000000000002)
+# CHECK-NEXT: Address range of this UnwindPlan: [stack-cfi-parsing.out..module_image + 0-2)
```

Comment:
`[stack-cfi-parsing.out..module_image + 0-2)`

`[...)` is an interval, `stack-cfi-parsing.out..module_image` is the start of the interval, and `+ 0-2` means it goes to, but does not include `stack-cfi-parsing.out..module_image - 2` ?

Or does it mean it's bytes 0 and 1, but not including 2, starting from the base address `stack-cfi-parsing.out..module_image`?

Not being hex is nice but it's not that easy to understand overall.

Reply:
The start of the interval is `.module_image + 0` and the end is `.module_image + 2` (`+2` is not included in the range). I agree it's not very clear, but I haven't come up with something better. I considered `[stack-cfi-parsing.out..module_image + 0, + 2)` but I'm not sure if that helps. `[stack-cfi-parsing.out..module_image + 0, stack-cfi-parsing.out..module_image + 2)` would be clear, but then it gets verbose again...

Reply:
Got it.

Well this is nic*er* than it was, so it's fine.